pkg/rulesConfig/defaultRules/10-ensure-sca-scanner.go

```diff
@@ -96,6 +98,15 @@ func githubErrorsRule10(githubData map[string]*githubConnector.GithubOwner) ([]*
 				}
 			}
 
+			if !foundScaScanner {
+				var err error
+				foundScaScanner, err = findJfrogScanScannerRule10(repo.JfrogPipelines, sharedRegexExpressions)
```

Comment:
Can't place the comment on line 114 - but the CiCdPlatform should now be either github/jfrog